pkg/dashboard/resource/project.go

```diff
@@ -271,6 +276,15 @@ func (pr *projectResource) createProject(projectInfoInstance *projectInfo) (id s
 	return
 }
 
+func (pr *projectResource) getRequestOriginAndSessionCookie(request *http.Request) (platformconfig.ProjectsLeaderKind, *http.Cookie) {
+	requestOrigin := platformconfig.ProjectsLeaderKind(request.Header.Get("x-projects-role"))
```

Comment:
```suggestion
	requestOrigin := platformconfig.ProjectsLeaderKind(request.Header.Get("x-nuclio-projects-role"))
```

Reply:
retain the nuclio namespace within any additional header

Reply:
also, we not using `iguazio.ProjectsRoleHeaderKey`?

Reply:
and dont forget to add that header to `https://github.com/nuclio/nuclio/blob/master/pkg/dashboard/server.go#L216`

Reply:
> retain the nuclio namespace within any additional header

On an offline discussion we've decided to make it `x-projects-role` to have one header to be used by iguazio/mlrun/nuclio